po/de.po

```diff
@@ -1035,7 +1078,7 @@ msgstr "Einstellungen"
 
 #: setzer/dialogs/preferences/preferences_viewgtk.py:53
 msgid "Build System"
-msgstr ""
+msgstr "Textsatzsystem"
```

Comment:
I just noticed that it is really annoying that there is no real German word for building in terms of programming. I found this on the Wikipedia side of LuaTeX, I think it fits here nicely.

Reply:
I don't think that fits here, can you leave it blank for now?